strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/config/StorageMultipartConfig.java

```diff
@@ -0,0 +1,50 @@
+package org.carlspring.strongbox.config;
+
+import org.carlspring.strongbox.config.util.UpdatableMultipartConfigElement;
+import org.carlspring.strongbox.filter.ArtifactSizeFilter;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.web.servlet.MultipartProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;;
+import org.springframework.boot.web.servlet.FilterRegistrationBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import javax.inject.Inject;
+import javax.servlet.MultipartConfigElement;
+
+@Configuration
+@EnableConfigurationProperties(MultipartProperties.class)
+public class StorageMultipartConfig
+{
+
+    @Inject
+    private MultipartProperties multipartProperties;
+
+    @Bean
+    @Qualifier("multipartConfigElement")
+    public MultipartConfigElement multipartConfigElement()
+    {
+        MultipartConfigElement multipartConfigElement = multipartProperties.createMultipartConfig();
+        return new UpdatableMultipartConfigElement(multipartConfigElement.getLocation(),
+                                                   multipartConfigElement.getMaxFileSize(),
+                                                   multipartConfigElement.getMaxRequestSize(),
+                                                   multipartConfigElement.getFileSizeThreshold());
+    }
+
+    @Bean
+    public ArtifactSizeFilter artifactSizeFilter()
```

Comment:
better to have all filters at `WebConfig`. Also have you tired to use `OncePerRequestFilter`? I mean maybe in this case we don't need `filteredMockMvc` in test?